pages/countries.js

```diff
@@ -201,24 +201,23 @@ class Countries extends React.Component {
                 <NavBar />
                 <RegionMenu>
                   <Container>
-                    <Box width={[1, 'unset']}>
+                    <Box ml={23} width={1}>
```

Comment:
It seems this `Box` doesn't have a purpose anymore. If removing it doesn't change anything, please remove it.

Reply:
Removed it